VultisigApp/VultisigApp/Services/CoinService.swift

```diff
@@ -96,6 +96,10 @@ struct CoinService {
             
             for token in tokens {
                 do {
+                    let existingCoin =  vault.coins.first{$0.chain == token.chain && $0.ticker == token.ticker}
```

Comment:
we're doing this check a lot. What you think about helper like: vault.isExist(coin: CoinMeta) -> Bool ?

Reply:
Good idea , we can add it

Reply:
`vault.coin(for: token)` , there is already one